bin/receiver.py

```diff
@@ -104,36 +104,26 @@ def main():
         print 'Error configuring logging: %s' % str(err)
         print 'SSM will exit.'
         sys.exit(1)
-        
+
     global log
     log = logging.getLogger('ssmreceive')
 
-    # Set defaults for MQ-BROKER only variables
-    use_ssl = None
-    # Set defaults for AMS only variables
-    project = None
-    token = ""
-
     log.info(LOG_BREAK)
     log.info('Starting receiving SSM version %s.%s.%s.', *__version__)
 
     # Determine the protocol and destination type of the SSM to configure.
```

Comment:
as there is no longer a destination type, this comment should be reworded

Reply:
Done.